microprofile/tests/tck/tck-lra/src/test/java/io/helidon/microprofile/lra/tck/CoordinatorDeployer.java

```diff
@@ -52,18 +54,21 @@ public void beforeStart(@Observes BeforeStart event, Container container) throws
         HelidonContainerConfiguration containerConfig = helidonContainer.getContainerConfig();
 
         containerConfig.addConfigBuilderConsumer(configBuilder -> {
-            configBuilder.withSources(MpConfigSources.create(CoordinatorService.class.getResource("/application.yaml")),
+            var is = CoordinatorService.class.getResourceAsStream("/application.yaml");
+            configBuilder.withSources(MpConfigSources.create(ConfigSources.create(is, MediaType.APPLICATION_X_YAML.toString())),
```

Comment:
why do you use X_YAML and not YAML?

Reply:
I need correct config parser to kick in https://github.com/oracle/helidon/blob/master/config/yaml/src/main/java/io/helidon/config/yaml/YamlConfigParser.java#L55